conf/app.routes

```diff
@@ -44,3 +44,5 @@ GET         /csv-download-error                     controllers.DownloadCsvContr
 
 GET         /transaction/:ref                       controllers.DeclarationDetailController.displayDetails(ref: String, page: Option[Int])
 GET         /transaction-search/:searchInput        controllers.DeclarationDetailController.displaySearchDetails(page: Option[Int], searchInput: String)
```

Comment:
Why this has not been updated to POST method (to hide the search parameter i.e. MRN and UCR) as we discussed?

Reply:
I had to revoke it back to GET as merge conflicts were quite messy.

Reply:
Okay, but have this been conveyed to Harsha to address it as part of separate ticket?

Reply:
It will be handled as part of separate ticket and has been conveyed to Harsha.